snap/snapcraft.yaml

```diff
@@ -0,0 +1,57 @@
+name: dvc
+summary: Data Version Control
+description: Git for Data & Models https://dvc.org
+adopt-info: dvc
+grade: stable
+confinement: strict
+base: core18
+license: Apache-2.0
+plugs:
+  config-dvc:
+    interface: system-files
+    read:
+    - /etc/hosts
+    - /etc/host.conf
+    - /etc/resolv.conf
+    write:
+    - /etc/dvc.conf
+parts:
+  dvc:
+    plugin: python
+    python-packages: [wheel]
+    source: .
+    build-packages:
+    - git
+    - libc-dev
+    stage-packages:
+    - git
+    override-pull: |
+        snapcraftctl pull
+        echo 'PKG = "snap"' > $SNAPCRAFT_PART_SRC/dvc/utils/build.py
+        snapcraftctl set-version `cd $SNAPCRAFT_PART_SRC && git describe --tags`
+        # install all optional extras
+        sed -ri 's/(=install_requires)/\1+all_remotes+hdfs/' $SNAPCRAFT_PART_SRC/setup.py
+    override-build: |
+        snapcraftctl build
+        cp $SNAPCRAFT_PART_BUILD/scripts/completion/dvc.bash $SNAPCRAFT_PART_INSTALL/completion.sh
```

Comment:
bash but no zsh?

Reply:
doesn't look possible https://forum.snapcraft.io/t/zsh-integration-for-the-snap-command/11094